services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/team/TeamManagerImpl.java

```diff
@@ -321,12 +318,63 @@ private void bindTeamName(String name, Long teamId){
 	}
 
 	@Override
-	public PaginatedResults<TeamMember> listMembers(String teamId, long limit,
+	public PaginatedResults<TeamMember> listMembers(String teamId, TeamMemberTypeFilterOptions memberType, long limit,
 			long offset) throws DatastoreException {
-		List<TeamMember> results = teamDAO.getMembersInRange(teamId, limit, offset);
+		List<TeamMember> results;
+		Set<Long> adminIds = teamDAO.getAdminTeamMemberIds(teamId).stream().map(Long::valueOf).collect(Collectors.toSet());
+		switch (memberType) {
+			case ADMIN:
+				if (adminIds.isEmpty()) { // No team admins is possible at the DAO level
+					return new PaginatedResults<>();
+				}
+				results = teamDAO.getMembersInRange(teamId, adminIds, null, limit, offset);
```

Comment:
add documentation to this method stating that isAdmin will not be set at the dao level